File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.py
class Solution(object):
    def sumZero(self, n):
        output = []
        for i in range(1, n // 2 + 1):
            output.append(i)
            output.append(-i)
        if n % 2 != 0:
            output.append(0)
        return output

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.java
class Solution 
{
    public int[] sumZero(int n) 
    {
        int output[]=new int[n];
        int start=0;
        int end=n-1;
        while(start<end)
        {
            output[start] = -1*end;
            output[end] = end;
            start++;
            end--;
        }
        return output;
    }
}